packages/jest-mock/tsconfig.json

```diff
@@ -0,0 +1,7 @@
+{
+  "extends": "../../tsconfig.json",
+  "compilerOptions": {
+    "rootDir": "src",
+    "outDir": "build"
+  }
```

Comment:
```suggestion
  },
  "references": [{"path": "../jest-types"}]
```

Reply:
Added. Interesting TS figured it out without the `references`. Wonder in which cases we need that actually

Reply:
we need https://github.com/Microsoft/TypeScript/issues/25376

Reply:
this is just for `tsc --build` to build things in the correct order, the actual type information doesn't depend on this

Reply:
noted